file_formats/fip.md

```diff
@@ -20,39 +20,51 @@ For example:
 
 ```plaintext
 📦 fib
-┣ FIP_DataG_2024-06-05T08_25_33.csv
-┣ FIP_DataIso_2024-06-05T08_25_33.csv
-┣ FIP_DataR_2024-06-05T08_25_33.csv
-┣ FIP_RawG_2024-06-05T08_25_33.bin
-┣ FIP_RawIso_2024-06-05T08_25_33.bin
-┣ FIP_RawR_2024-06-05T08_25_33.bin
-┣ FIP_ROIsG-Iso_2024-06-05T08_25_33.csv
-┗ FIP_ROIsR_2024-06-05T08_25_33.csv
+┣📂 SoftwareEvents
+┃ ┣📜RepositoryStatus.json
+┃ ┗📜Region.json
+┣ green.csv
+┣ red.csv
+┣ iso.csv
+┣ green.bin
+┣ red.bin
+┣ iso.bin
+┣ roi_green_iso.csv
+┣ roi_red.csv
+┣ camera_green_iso_metadata.csv
+┗ camera_red_metadata.csv
 ```
-The file names are `FIP_[channel-name]_[datetime].csv`.  The `[channel-name]` token can be one of the following:
 
-* `DataG`: data for green channel
-* `DataR`: data for red channel
-* `DataIso`: data for isosbestic channel
+#### Photometry CSV files
 
-#### Photometry Readout CSV files
+Data files are named by the color of the channel.
+
+* `green.csv`: data for green channel
+* `red.csv`: data for red channel
+* `iso.csv`: data for isosbestic channel
 
 These files contain photometry readouts. Each row describes the average signal of pixels within each ROI for a single video frame. The values are computed online during data acquisition. The files have no headers. Each column is a timeseries, ordered as follows:
 
-* `timestamp` (software timestamp, in milliseconds, total time of the day)
+* `SoftwareTS` (software timestamp, in milliseconds, total time of the day)
 * `ROI0` (corresponding to fiber branch1) values
 * `ROI1` (corresponding to fiber branch2) values
 * `...`  (depending on how many fibers are used; in most of the experiments: ROI0-3/4branches)
-* `Blank ROI`: CMOS dark count floor signal
+* `Background`: CMOS dark count floor signal
+* `HarpTS` (Harp timestamp, in XXXunit, from Harp device reference)
```

Comment:
- SoftwareTimestamp
- HarpTimestamp
- Fiber_0
- ...
- Fiber_N

There should be a header row with these strings in them. Indicate users should use these for metadata lookups.

@bruno-f-cruz I think this pattern addresses your casing concerns.

Reply:
That's fine. I would just ask to consider using the same nomenclature as the behavior videos for the common fields. https://github.com/AllenNeuralDynamics/aind-file-standards/blob/main/file_formats/behavior_videos.md

At the end of the day they are doing the same thing - aligning frames to triggers - no need to come up with yet another way to call the same concept.

Reply:
I forgot to explicitly state that these are headers. I will look at @bruno-f-cruz documentation and explicitly state that these are the headers in the file

Reply:
Oh wow, just realized I left in 'these values have not headers". That was an error on my part

Reply:
@arielleleon 
Since there were no headers previously. Why not just refactor to use the header names in the behavior-video standards? See my original response to David.

Reply:
Yes - sorry @bruno-f-cruz, I keep getting distracted but I am in the process of synchronizing the names here to your behavior - video standards